src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/tests/Input.test.js

```diff
@@ -40,6 +40,14 @@ test('Input should call the callback when the input changes', () => {
     expect(onChange).toHaveBeenCalledWith('my-value');
 });
 
+test('Input should call the callback when icon was clicked', () => {
+    const onChange = jest.fn();
+    const handleIconClick = jest.fn();
+    const input = mount(<Input icon="my-icon" value="My value" onChange={onChange} onIconClick={handleIconClick} />);
```

Comment:
The test above tests something similar and works without `mount`. Why is it necessary here?